src/microbe_stage/IEngulfable.cs

```diff
@@ -9,6 +9,11 @@ public interface IEngulfable : IGraphicalEntity
     /// <summary>
     ///   The engulf size of this engulfable object.
     /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     This can also be made to scale with <see cref="DigestedAmount"/> for richer gameplay depth.
```

Comment:
I'd reword this that some classes may scale this. Right now this sounds like some future thing but seems to be already implemented in FloatingChunk.

Reply:
Doesn't this have similar meaning?